irrd/utils/text.py

```diff
@@ -97,3 +103,46 @@ def _str_to_camel_case(snake_str: str):
 
 def clean_ip_value_error(value_error):
     return re.sub(re_clean_ip_error, "", str(value_error))
+
+
+def dummify_object_text(rpsl_text: str, object_class: str, source: str, pk: str):
+    """
+    Modify the value of attributes in an RPSL object.
+    """
+
+    if not rpsl_text:
+        return rpsl_text
+
+    nrtm_response_dummy_object_class = get_nrtm_response_dummy_object_class_for_source(source)
+    if nrtm_response_dummy_object_class:
+        if object_class in nrtm_response_dummy_object_class:
+            dummy_attributes = get_setting(f"sources.{source}.nrtm_response_dummy_attributes")
+            if dummy_attributes:
+                if get_setting(f"sources.{source}.nrtm_response_dummy_remarks"):
+                    dummy_remarks = textwrap.indent(
+                        get_setting(f"sources.{source}.nrtm_response_dummy_remarks"), "remarks:".ljust(16)
```

Comment:
Where does that 16 come from? From somewhere in irrd.rpsl right? Feels like we should make that a constant then.

Reply:
yes, it comes from RPSL_ATTRIBUTE_TEXT_WIDTH which has been used in different places. Updated to use the constant, thanks